packages/binding-coap/src/coap-client.ts

```diff
@@ -44,9 +44,6 @@ export default class CoapClient implements ProtocolClient {
         // WoT-specific content formats
         coap.registerFormat(ContentSerdes.JSON_LD, 2100);
         // TODO also register content fromat with IANA
-        // from experimental range for now
-        coap.registerFormat(ContentSerdes.TD, 65100);
-        // TODO need hook from ContentSerdes for runtime data formats
```

Comment:
Hmm, should this TODO be kept?

Reply:
Good question, not really sure. Work was mainly driven by @mkovatsc at that time.